test/inductor/extension_backends/extension_device.cpp

```diff
@@ -116,6 +116,20 @@ at::Tensor custom__copy_from(const at::Tensor& self, const at::Tensor& dst, bool
   return dst;
 }
 
+at::Tensor custom_empty_memory_format(at::IntArrayRef size,
```

Comment:
I copied this from `test/cpp_extensions/open_registration_extension.cpp`. Not sure why we have a bunch of definitions duplicated between these two files...